plugin/manager_linux.go

```diff
@@ -183,8 +183,8 @@ func (pm *Manager) upgradePlugin(p *v2.Plugin, configDigest digest.Digest, blobs
 				return
 			}
 
-			if err := os.Rename(backup, orig); err != nil {
-				err = errors.Wrap(err, "error restoring old plugin root on upgrade failure")
```

Comment:
But `err` is masked here, or did I misunderstand your comment? https://play.golang.org/p/rDRlWDXr1J